torch/_dynamo/eval_frame.py

```diff
@@ -413,13 +413,9 @@ def toy_example(a, b):
     if disable or os.environ.get("TORCHDYNAMO_DISABLE", "") == "1":
         return _NullDecorator()
     if sys.platform == "win32":
-        warnings.warn(
-            "Windows is not currently supported, torch.compile() will do nothing"
-        )
-        return _NullDecorator()
+        raise RuntimeError("Windows not yet supported for torch.compile")
```

Comment:
Nit: Should be NotImplementedError?

Reply:
I would imagine NotImplementedError is typically used for unimplemented methods in abstract base classes, not for missing features.